utils/build-script-impl

```diff
@@ -2249,10 +2249,10 @@ if [[ "${INSTALLABLE_PACKAGE}" ]] ; then
       fi 
 
       (cd "${INSTALL_DESTDIR}" &&
-        tar -c -z -f "${INSTALLABLE_PACKAGE}" "${TOOLCHAIN_PREFIX/#\/}")
+        tar -c -J -f "${INSTALLABLE_PACKAGE}" "${TOOLCHAIN_PREFIX/#\/}")
```

Comment:
This is a branch specific to OS X.  `tar` on OS X does not support the `-J` flag.